src/renderer/reader/components/Reader.tsx

```diff
@@ -777,7 +831,7 @@ class Reader extends React.Component<IProps, IState> {
             return;
         }
 
-        this.navLeftOrRight_(isPrevious, true);
```

Comment:
This seems wrong ... the whole point of `onKeyboardSpineNavigationPreviousNext()` is that the navigation should be at the spine level, not at the internal pagination or scrolling level. @panaC do you remember why you introduced this change in logic?

Reply:
Rectified this via https://github.com/edrlab/thorium-reader/pull/1188/commits/60187f714dfe645c98b7f1afc7a533c44f1d403d

...but still would like to understand the rationale for the code modification, just in case there was a utility to it in the context of this PDF PR.